com.openwall.John.json

```diff
@@ -0,0 +1,94 @@
+{
+    "id": "com.openwall.John",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "1.6",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "john",
+    "tags": [ "1.9J1-4bf9652", "1.9.0 Jumbo 1" ],
+    "finish-args": [
+        "--filesystem=home"
```

Comment:
does the app only requires home access, no gui or anything else?

Reply:
Yes.